nautobot/extras/forms/forms.py

```diff
@@ -196,6 +196,7 @@ class Meta:
         fields = (
             "content_type",
             "label",
+            "grouping",
```

Comment:
Probably should add `grouping` as a field to `ComputedFieldFilterSet` as well... though I see it isn't included in `CustomFieldFilterSet` either, maybe add it there too. :-)

Reply:
Updated